vendor/src/github.com/docker/distribution/digest/verifiers.go

```diff
@@ -1,8 +1,6 @@
 package digest
```

Comment:
Its out of scope for this PR as it doesn't apply for this use case. But generally, a cryptographic verifier should be constant time.

I'll submit a PR.